OutOfSchool/OutOfSchool.DataAccess/Models/Workshop.cs

```diff
@@ -94,8 +94,6 @@ public class Workshop : BusinessEntity, IImageDependentEntity<Workshop>, IHasEnt
     [Required(ErrorMessage = "Type of pay rate is required")]
     public PayRateType PayRate { get; set; }
 
-    public bool WithDisabilityOptions { get; set; } = default;
-
     [MaxLength(200)]
     public string DisabilityOptionsDesc { get; set; } = string.Empty;
```

Comment:
You need to also remove `DisabilityOptionsDesc` as this field is tied to `WithDisabilityOptions` :)